recipe/bld.bat

```diff
@@ -33,6 +33,8 @@ cmake -LAH -G "NMake Makefiles" ^
       -DENABLE_NCZARR_S3=on ^
       -DENABLE_NCZARR_S3_TESTS=on ^
       -DENABLE_S3_SDK=on ^
```

Comment:
peanut gallery here, doesn't this need to be turned off?

Reply:
Yeah I was wondering about that. It wasn't clear to me so I tried this first. But that didn't work. Feel free to try ; otherwise I will do it tomorrow.